pipelines/py/bump-constraints.yaml

```diff
@@ -0,0 +1,58 @@
+name: Bump Python constraint pins
+
+inputs:
+  constraints-file:
+    description: Path to the constraints file to update
+    default: constraints.txt
+  packages:
+    description: List of package==version pairs to update (newline separated, with optional comments)
+    required: true
+  only-replace:
+    description: Only update packages already in the constraints file (set to false to allow adding new packages)
+    default: "true"
+  show-diff:
+    description: Show diff of changes made
+    default: "false"
+
+needs:
+  packages:
+    - tw
+
+pipeline:
+  - name: "bump constraint pins"
+    runs: |
+      set -euo pipefail
+
+      constraints_file="${{inputs.constraints-file}}"
+      packages="${{inputs.packages}}"
+      only_replace="${{inputs.only-replace}}"
+      show_diff="${{inputs.show-diff}}"
+
+      # Check if constraints file exists
```

Comment:
you can just drop this here.  your go code will catch it and say 'constraints file '%s' not found". less shell is better.